packages/visualizations/app/styles/navi-visualizations/components/visualization-config/metric-label.less

```diff
@@ -5,30 +5,20 @@
 
 .metric-label-config {
   .config-container();
-}
 
-.metric-label-config__format-list {
-    display: flex;
-    flex-direction: column;
+  .number-format-selector__section-header {
+    .metric-label-config__section-header;
+  }
```

Comment:
extra `}`

Reply:
Why `}` is extra?

Reply:
oh never mind, the git diff made it look like line 12 had an extra `}`